tests/driver_dac_dds/main.c

```diff
@@ -312,7 +329,9 @@ static const shell_command_t shell_commands[] = {
 
 int main(void)
 {
-    dac_init(DAC_DDS_PARAM_DAC);
+    dac_init(DAC_DDS_DAC);
+    /* Initialize to the idle level of 16bit audio */
+    dac_set(DAC_DDS_DAC, 1 << 15);
```

Comment:
maybe `dac_dds_init()` should take care of that?

Reply:
Possibly -- but the question would come back with dac_init and dac_poweron/poweroff, and eventually to the question of turning on and off the power amplifier. Leaving dac_dds to be a tool that feeds a DAC with whichever values are passed in seems to be a clean cut of responsibilities -- every part that implies that the DAC data is audio would be left in the application.